pkg/didmethod/peer/resolver.go

```diff
@@ -12,12 +12,13 @@ import (
 
 // DIDResolver resolver
 type DIDResolver struct {
-	store *DIDStore
+	store  *DIDStore
+	method string
```

Comment:
In the case of peer DIDs the method is always "peer", so this can be hard coded to `peer.DIDResolver.Accept()`

Reply:
done